OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -80,7 +80,7 @@ public class Cargo : IPips, IIssueOrder, IResolveOrder, IOrderVoice, INotifyCrea
 	{
 		public readonly CargoInfo Info;
 		readonly Actor self;
-		readonly Stack<Actor> cargo = new Stack<Actor>();
+		readonly List<Actor> cargo = new List<Actor>();
```

Comment:
I assume the intention here was to make this a list so that you can remove an arbitrary actor, but that isn't actually implemented in the current PR version.